store/sqlstore/upgrade.go

```diff
@@ -1593,5 +1595,21 @@ func hasMissingMigrationsVersion600(sqlStore *SqlStore) bool {
 		return true
 	}
 
+	scIdInfo, err := sqlStore.GetColumnInfo("Posts", "FileIds")
+	if err != nil {
+		mlog.Error("Error getting column info for migration check",
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.